studio/deskStructure.js

```diff
@@ -1,22 +1,18 @@
 import S from '@sanity/desk-tool/structure-builder'
```

Comment:
This pull request could've been split into two. One for changes in `studio/` and one for design changes. It makes it easier to review.

Reply:
Thank you, in the next PRs I will make them separately